### ISSUE TITLE
Create JavaScript Debug Terminal fails with "[...] createDebuggerTerminal not found"

### ISSUE THREAD

**Issue by jonasb** [microsoft/vscode#116106]
- VSCode Version: Code - Insiders 1.54.0-insider (afd102cbd2e17305a510701d7fd963ec2528e4ea, 2021-02-08T05:23:20.010Z)
- OS Version: Darwin arm64 20.3.0

Steps to Reproduce:

1. Select "Create JavaScript Debug Terminal"
2. Error "command 'extension.pwa-node-debug.createDebuggerTerminal' not found" show up. No terminal is launched.

Does this issue occur when all extensions are disabled?: Yes

Issue Type: <b>Bug</b>

Select "Create JavaScript Debug Terminal"
Error "command 'extension.pwa-node-debug.createDebuggerTerminal' not found" show up

VS Code version: Code - Insiders 1.54.0-insider (afd102cbd2e17305a510701d7fd963ec2528e4ea, 2021-02-08T05:23:20.010Z)
OS version: Darwin arm64 20.3.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|undefined|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|2, 2, 2|
|Memory (System)|16.00GB (0.06GB free)|
|Process Argv|--crash-reporter-id 55ee016c-c75e-493b-a74d-a1848952247e|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (6)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-eslint|dba|2.1.14
vscode-npm-script|eg2|0.3.13
prettier-vscode|esb|5.9.1
mdx|sil|0.1.0
code-spell-checker|str|1.10.2
shellcheck|tim|0.13.1

</details><details>
<summary>A/B Experiments</summary>

```
vsliv695:30137379
vsins829:30139715
vsliv368:30146709
vsreu685:30147344
python383:30185418
vspyt653:30253241
vspor879:30202332
vspor708:30202333
vspor363:30204092
vswsl492:30208929
vstry244:30244315
pythonvsdeb440:30224570
pythonvsded773:30223139
pythonvspyt875:30251590
core-portspanel:30233467
coreuntitledfile:30249963

```

</details>

**Comment by jonasb** [comment]
Worked yesterday before I updated to the latest insider version.

**Comment by dbaeumer** [comment]
Same for me. Breaks my whole debugger flow :-(

**Comment by connor4312** [comment]
Fixed in 86701b150d2
